nl/mail/chrome/messenger/updates/pbUpdates.dtd

```diff
@@ -1,17 +1,17 @@
 <!ENTITY releaseNotes.label "Release Notes:">
-<!ENTITY cancelButton.label "Cancel">
-<!ENTITY installUpdateButton.label "Install Update">
-<!ENTITY newVersion.label "A new version of Postbox is available!">
+<!ENTITY cancelButton.label "Annuleren">
+<!ENTITY installUpdateButton.label "Installeer Update">
+<!ENTITY newVersion.label "Er is een nieuwe versie van Postbox beschikbaar">
 
-<!ENTITY updatingPostbox.label "Updating Postbox">
-<!ENTITY downloading.label "Downloading update...">
+<!ENTITY updatingPostbox.label "Postbox updaten">
+<!ENTITY downloading.label "Update downloaden...">
 
-<!ENTITY errorTitle.label "Error">
-<!ENTITY errorProse.description "An error occured during the update. Please download and install the latest version of Postbox from our website.">
-<!ENTITY continueButton.label "Continue">
+<!ENTITY errorTitle.label "Fout">
+<!ENTITY errorProse.description "Er is een fout opgetreden tijdens de update. Download en installeer de nieuwste versie van Postbox van onze website.">
```

Comment:
CORRECTION NEEDED, it should be: "Er is een fout opgetreden tijdens de update. Download en installeer de nieuwste versie van Postbox op onze website."